.gitignore

```diff
@@ -21,7 +21,7 @@ coverage/
 public/dist/
 uploads
 modules/users/client/img/profile/uploads
-config/env/local.js
+config/env/local*.js
```

Comment:
Good one, but because local.example.js is already committed to the repository before this change was introduced then git won't ignore it (even though the regex says otherwise).